apps/transport/lib/transport/feedback.ex

```diff
@@ -0,0 +1,56 @@
+defmodule DB.Feedback do
+  @moduledoc """
+  Stores feedback from users about the application sent through the feedback form
+  """
+  use Ecto.Schema
+  use TypedEctoSchema
+  import Ecto.Changeset
+
+  @features [
+    :gtfs_stops,
+    :on_demand_validation,
+    :gbfs_validation,
+    :reuser_space
+  ]
+
+  @ratings [:like, :neutral, :dislike]
+
+  schema "feedback" do
```

Comment:
```suggestion
  typed_schema "feedback" do
```
https://hexdocs.pm/typed_ecto_schema/TypedEctoSchema.html